src/Bridge/Symfony/Bundle/SonataTwigExtensionsBundle.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace Sonata\TwigExtensions\Bridge\Symfony\Bundle;
```

Comment:
Should we use `TwigExtensions` or `Twig` as namespace? Otherwise there is a `SonataTwigExtensionsExtension` if you create a symfony bundle.

Reply:
you can use any name and specify inside your `Bundle` file which extension to use if the only issue is the Extension. If there are other issues with this namespace we can change it. Hopefully I will pick this up soon and try to finish it.